app/Listeners/WebhookReceivedListener.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Listeners;
+
+use Illuminate\Support\Facades\Log;
+use Laravel\Paddle\Events\WebhookReceived;
+
+class WebhookReceivedListener
+{
+    /**
+     * Handle Paddle webhook event.
+     *
+     * @param  WebhookReceived  $event
+     * @return void
+     */
+    public function handle(WebhookReceived $event)
```

Comment:
How will we deal with that in production?

Reply:
This is superseeded by https://github.com/monicahq/customers/pull/42